apps/backend/listing/src/index.ts

```diff
@@ -1,25 +1,49 @@
 import express, { Request, Response, NextFunction } from "express";
 import morgan from "morgan";
-
-import { run } from "../../lib/src/example";
+import pgPromise from "pg-promise";
+import { getListingById } from "./getListingById";
+import { createListing } from "./createListing";
+import { updateListing } from "./updateListing";
+import { deleteListing } from "./deleteListing";
 
 const PORT = 8212;
 
 const app = express();
+
 app.use(morgan("dev"));
 app.use(express.json());
 
+const pgp = pgPromise();
+const DB_ENDPOINT = process.env.DB_ENDPOINT || "";
+
+if (!DB_ENDPOINT && process.env.NODE_ENV !== "test") {
```

Comment:
why is the env being checked here? It should either not be checked or the error message should reflect the conditions being checked